src/sentry/integrations/utils/codecov.py

```diff
@@ -20,12 +21,14 @@ def get_codecov_data(
         url = CODECOV_URL.format(
             service=service, owner_username=owner_username, repo_name=repo_name
         )
-        params = {"branch": branch, "path": path}
-        response = requests.get(
-            url, params=params, headers={"Authorization": f"tokenAuth {codecov_token}"}
-        )
-        response.raise_for_status()
-        line_coverage = response.json()["files"][0]["line_coverage"]
-        codecov_url = response.json()["commit_file_url"]
+        with configure_scope() as scope:
+            scope.set_tag("codecov.attempted_url", url)
```

Comment:
In order to get this code landed sooner could you please follow up in setting the tag within `set_tags` based on values from `result`? This has no rush.

I want to centralize where we set the tags. I want to move more things there but I have not had a chance.

Reply:
Good idea, I'll put up a PR for that change once this is merged